src/status_im2/subs/chat/chats.cljs

```diff
@@ -140,6 +141,24 @@
  (fn [[chat-id inputs]]
    (get inputs chat-id)))
 
+(re-frame/reg-sub
+ :chats/composer-height
+ :<- [:chats/current-chat-input]
+ :<- [:chats/link-previews-unfurled]
+ (fn [[{:keys [input-content-height metadata]} link-previews]]
+   (let [{:keys [responding-to-message editing-message sending-image]} metadata]
+     (+ (max composer.constants/input-height input-content-height)
+        (when responding-to-message
+          composer.constants/reply-container-height)
+        (when editing-message
+          composer.constants/edit-container-height)
+        (when (seq sending-image)
+          composer.constants/images-container-height)
+        (when (seq link-previews)
+          composer.constants/links-container-height)
+        composer.constants/bar-container-height
+        composer.constants/actions-container-height))))
+
```

Comment:
Did you include the text height inside the composer somewhere?

Reply:
isn't it `input-content-height`? (line 150)